validator/testcases/javascript/instanceactions.py

```diff
@@ -166,6 +171,11 @@ def setAttribute(args, traverser, node, wrapper):
             description="To prevent vulnerabilities, event handlers (like "
                         "'onclick' and 'onhover') should always be defined "
                         "using addEventListener.",
+            signing_help="Please use `addEventListener` any place you might "
+                         "otherwise create event listener attributes. Event "
+                         "listener attributes will not be accepted in add-ons "
+                         "submitted for automated signing in any instance "
+                         "where they may be reasonably avoided.",
```

Comment:
I can't think of any good examples off hand, and I think that they'll rarely get a pass, but I know I've come across usages in the past that I'd probably approve. The nearest example I can think of is copying event handler attributes from one element to another, within the same document.